core/client/src/main/java/alluxio/client/file/BaseFileSystem.java

```diff
@@ -325,4 +325,11 @@ public void unmount(AlluxioURI path, UnmountOptions options)
       mFileSystemContext.releaseMasterClient(masterClient);
     }
   }
+
+  @Override
+  public void close() {
+    if (mFileSystemContext != FileSystemContext.INSTANCE) {
+      mFileSystemContext.close();
```

Comment:
technically, it is possible to use one context to create multiple file system instances, right? In this case, will we have trouble if one file system instance is closed, causing the shared context also closed?